client/a8c-for-agencies/sections/team/primary/team-list/index.tsx

```diff
@@ -91,40 +123,37 @@ export default function TeamList() {
 				enableSorting: false,
 			},
 		],
-		[ isDesktop, translate ]
+		[ handleAction, isDesktop, translate ]
 	);
 
-	// FIXME: Fetch team members
-	const members: TeamMember[] = [
-		{
-			displayName: 'Owner',
-			email: 'owner@automattic.com',
-			role: 'owner',
-			status: 'active',
-		},
-		{
-			displayName: 'User 1',
-			email: 'user1@automattic.com',
-			role: 'member',
-			status: 'active',
-			dateAdded: new Date().toDateString(),
-		},
-		{
-			displayName: 'User 2',
-			email: 'user2@automattic.com',
-			role: 'member',
-			status: 'pending',
-			dateAdded: new Date().toDateString(),
-		},
-		{
-			email: 'user3@automattic.com',
-			role: 'member',
-			status: 'expired',
-		},
-	];
+	const members: TeamMember[] = useMemo( () => {
+		const activeMembers: TeamMember[] = [
+			// FIXME: Fetch team members
+			{
+				id: 0,
+				displayName: 'Owner',
+				email: 'owner@automattic.com',
+				role: 'owner',
+				status: 'active',
+			},
```

Comment:
Should we remove this dummy member?

Reply:
I will remove this on a separate PR when I integrate the users' endpoint.